tests/test_app.py

```diff
@@ -0,0 +1,82 @@
+from todo_app.data import session_items as si
+from todo_app.classes import  ViewModel
+from todo_app.app import create_app
+from dotenv import find_dotenv,load_dotenv
+import datetime as dt
+import pytest
+import json
+
+class TestUnit:
+    @staticmethod
+    def test_to_do_items():
```

Comment:
These tests depend on real environment variables set up and suitable data in the board in Trello.

Tests should not depend on real data and only end to end tests should be making requests to Trello.

Also, it's not clear what you are trying to test - this test is dependent on both `session_items` and view_model` to behave correctly. But not that the app integrates the two of them correctly.

The exercise asked you to create *unit* tests for ViewModel. Which means you should just hardcode some `items` data for these tests and then check that ViewModel behaves correctly.

Reply:
I  stored the items variable using pickle and stored/commited the file. This way I always load that set of items.